Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -27,110 +27,95 @@ Your shape
 #include<string>
 #include <cmath>
 
-class Square
+class FSquare
 {
 private:
-	float Side;
-	float AreaSquare;
-	float PerimeterSquare;
+	float SideLenght;
 public:
-	Square()
+	FSquare()
 	{
+		SideLenght = 0.0f;
 	}
-	Square(float Radius, float Diameter)
-	{
-		Side = 0;
-	}
-	void DataSquare()
+	
+	
+	void InputDataSquare()
 	{
-		float side;
+		float EntrySide;
 		std::cout << "what is the side of Your Square:" << std::endl;
-		std::cin >> side;
+		std::cin >> EntrySide;
 
-		while (!std::cin.good() || side < 0)
+		while(!std::cin.good() || EntrySide < 0.0f)
 		{
 			std::cout << "Invalid input, please enter The side of Your square again" << std::endl;
 			std::cin.clear();
 			std::cin.ignore(1000, '\n');
-			std::cin >>side;
+			std::cin >>EntrySide;
 		}
-		Side = side;
+		SideLenght = EntrySide;
 	}
-	float GetAreaSquare( )
+	float GetAreaSquare()
```

Comment:
Your methods should do what they _imply_ they do and should not have side effects. These methods are `getter` methods, meaning that when you read this method you expect them to return a value, not print anything to the screen. With that in mind, these getter methods should not be the ones responsible for printing anything to the screen, both for this shape and the circle shape.